src/lua/init.lua

```diff
@@ -194,15 +211,31 @@ local function search(name)
 end
 
 -- loader_preload 1
-table.insert(package.loaders, 2, cwd_loader_func(false))
-table.insert(package.loaders, 3, cwd_loader_func(true))
-table.insert(package.loaders, 4, rocks_loader_func(false))
-table.insert(package.loaders, 5, rocks_loader_func(true))
--- package.path   6
--- package.cpath  7
--- croot          8
+table.insert(package.loaders, 2, get_loader_func({
+    search_fn = get_search_func({ path_fn = fio.cwd, templates = LUA_TEMPLATES }),
+    load_fn = load_lua }))
+table.insert(package.loaders, 3, get_loader_func({
+    search_fn = get_search_func({ path_fn = fio.cwd, templates = LIB_TEMPLATES }),
+    load_fn = load_lib }))
```

Comment:
Rocks are searched in approot first (if set), but regular packages are searched first in cwd. Looks inconsistent.

Reply:
Made it more consistent by merge cwd loader with searchroot loader. Now if no searchroot defined, it acts as simple cwd loader